content/en/docs/components/pipelines/author-a-pipeline/components.md

```diff
@@ -165,52 +165,33 @@ To create a containerized component, you must:
 
 ### 3. Custom container components
 
-Container-based components make it easier to use the pre-existed container to build a KFP component. 
-The user will control the container entrypoint and is able to define the component by specifying their container, commands, and args in a `ContainerSpec` object with similar authoring experiences as defining Python components. 
+Custom container components make it easier to use the pre-existed container to build a KFP component. 
+You can control the container entrypoint and define the custom container component by specifying their image, commands, and args in a `ContainerSpec` object with similar authoring experiences as defining Python components. 
 
 To define a container component, you must:
-1) Write your component’s code as a python function that returns a `dsl.ContainerSpec` object to specify the container image and the commands to be run in the container and wrap the function into a `@container_component` decorator. The function should satisfy the following requirements:
-
-    *   It should not use any code declared outside of the function definition.
-    *   The function should do nothing other than returning a `dsl.ContainerSpec`
-        object, with the following parameters:
-        * image: The target image where the command (with args) is executed. The user will control the entrypoint of the image.
-        * command(optional): The command to be executed. 
-        * args(optional): The arguments of the command. It’s recommended to place the input of the     components in the args section instead of the command section.
+1) Write your component’s code as a Python function that returns a `dsl.ContainerSpec` object to specify the container image and the commands to be run in the container and wrap the function into a `@container_component` decorator. The function should do nothing other than returning a `dsl.ContainerSpec` object, with the following parameters:
+    * image: The target image where the command (with args) is executed. You will control the entrypoint of the image.
```

Comment:
I suggest a slight rephrasing. The command is executed in the _container_, which runs an image. And you _may_ control the entrypoint, but do not need to (optional args).

```suggestion
    * image: The image that the container will run. You can use `command` and `args` to control the entrypoint.
```